insights-inventory-client/src/main/java/org/candlepin/subscriptions/inventory/client/InventoryServiceProperties.java

```diff
@@ -31,6 +31,7 @@
 @Setter
 public class InventoryServiceProperties {
   private boolean useStub;
+  @Getter @Setter private boolean prettyPrintJson;
```

Comment:
There is already `@Getter` and `@Setter` on this class...are those not working for this attribute?